static_src/actions/service_actions.js

```diff
@@ -146,10 +147,13 @@ const serviceActions = {
     return Promise.resolve(serviceInstance);
   },
 
-  errorCreateInstance(err) {
+  errorCreateInstance(error) {
+    const { response } = error;
+    const safeError = (response && response.data) || { code: 500 };
```

Comment:
coming from a non-JS perspective:

it looks like `(response && response.data)`  could evaluate to true or false.
if true, safeError would be set to true.

if false, it would set safeError to {code: 500};

Now your tests, tell me it doesn't work that way, but just could be a little confusing at first glance.